db/changes.go

```diff
@@ -171,22 +171,27 @@ func (db *Database) AddDocInstanceToChangeEntry(entry *ChangeEntry, doc *Documen
 	}
 }
 
-func (db *Database) buildRevokedFeed(singleChannelCache SingleChannelCache, options ChangesOptions, revokedSeq, revocationSinceSeq uint64, to string) <-chan *ChangeEntry {
+// Parameters
+// revokedAt: This is the point at which the channel was revoked from the user. Used here for the triggeredBy value of
+// the.
+// revocationSinceSeq: The point in time at which we need to 'diff' against in order to check what documents we need to
+// revoke. This is used in this function in 'wasDocInChannelAtSeq'.
```

Comment:
Is it correct to add something like this:
```suggestion
// revoke - only documents in the channel prior to revocationSinceSeq may have been replicated, and need to be revoked. This is used in this function in 'wasDocInChannelAtSeq'.
```

Reply:
I think more like `documents in the channel **at** revocationSinceSeq may have been replicated, and need to be revoked` but yeah.

Reply:
I don't understand what's meant by **at** here, can you clarify?

Reply:
Well the point at which this value is used is in `wasDocInChannelAtSeq` in the case where the document has been mutated (or moved into channel AFTER the revocation). We need to validate whether that document was in the channel when we had access to it. `revocationSinceSeq` represents the last time we pulled based from since. 
So basically we're saying "Only revoke documents that were in the channel at this sequence". In my head "prior" may include channels that have already been previously revoked.